tango_simlib/tango_sim_generator.py

```diff
@@ -458,7 +460,7 @@ def get_parser_instance(sim_datafile):
         parser_instance.parse(sim_datafile)
     return parser_instance
 
-def configure_device_model(sim_data_file=None, test_device_name=None):
+def configure_device_models(sim_data_file=None, test_device_name=None):
```

Comment:
Changing this public API will affect users with existing device servers.  Consider keeping the old function as well, for backwards compatibility.  It could use the new function and raise an exception if there are multiple models.

Reply:
Thanks for pointing this out.